translations/lxqt-runner.ts

```diff
@@ -5,51 +5,61 @@
     <name>ConfigureDialog</name>
     <message>
         <location filename="../configuredialog/configuredialog.ui" line="14"/>
+        <location filename="../build/lxqt-runner_autogen/include/ui_configuredialog.h" line="154"/>
```

Comment:
All this _build/..._ locations shouldn't be included.

Reply:
oops, should be ok now.